shipyard-controller/debug-ui/fetch_events.js

```diff
@@ -0,0 +1,75 @@
+fetchEvents();
+
+function fetchEvents() {
+  const urlParams = new URLSearchParams(window.location.search);
+  const context = urlParams.get("shkeptncontext");
+  const projectname = urlParams.get("projectname");
+
+  let list = document.getElementById("tree_container");
+  list.innerHTML = "";
+
+  fetch(
+    "../sequence/project/" + projectname + "/shkeptncontext/" + context + "/event",
```

Comment:
the `..` makes it harder to reuse the script. Maybe it would be better to introduce a basepath value🤔